app/controllers/feeds_controller.rb

```diff
@@ -2,6 +2,37 @@ class FeedsController < ApplicationController
 
   def index
     @user = User.find_by(id: session[:user_id])
-    
+    if @user && @user.instagrams
+      @response = []
+      @user.instagrams.each do |gram|
+        @response << HTTParty.get(INSTAGRAM_URI + "#{gram.provider_id}/media/recent?access_token=#{session[:access_token]}")
+      end
+    end
+
+    if @user && @user.tweets
+      @people = []
+      # @people << Instagram.find(@user.instagram_ids)
+      @people << Tweet.find(@user.tweet_ids)
+      @people.flatten!
```

Comment:
If I'm following this code right, what you're most interested in here is the `username` attribute (line 22). If that's the case, we can simplify this block of code to just grab usernames without having to do a lot of Enumerable work. This is untested, so it may need some tweaking, but my first inclination is to write something like...

```ruby
# we can call @user.tweets thanks to the has_and_belongs_to_many relationship
usernames = @user.tweets.map &:username
```